guides/source/configuring.md

```diff
@@ -1009,6 +1017,14 @@ production:
   prepared_statements: false
 ```
 
+Advisory Locks are enabled by default on PostgreSQL and are used to make database migrations concurrent safe. You can disable advisory locks by setting `advisory_locks` to `false`:
+
+```yaml
+production:
+  adapter: postgresql
+  advisory_locks: false
+```
+
 If enabled, Active Record will create up to `1000` prepared statements per database connection by default. To modify this behavior you can set `statement_limit` to a different value:
```

Comment:
This is a continuation of the preceding discussion of `prepared_statements`.

More generally, the above feels a bit heavy-weight for what should be _relatively_ obscure configuration, to me. How about we combine the two: "By default Active Record uses database features like prepared statements and advisory locks. You might need to disable those features if you're using an external connection pooler like PgBouncer: [yaml]"?